public-ui/src/app/views/quick-search/quick-search.component.ts

```diff
@@ -202,16 +213,24 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
   public viewEhrDomain(r) {
     localStorage.setItem('ehrDomain', JSON.stringify(r));
     localStorage.setItem('searchText', this.prevSearchText);
-    this.router.navigateByUrl('ehr/' + r.domain.toLowerCase().replace(' ', '-' ));
+    for (const item in this.dbc.routeToDomainMap) {
```

Comment:
Instead of having for each multiple times, could we have two hashmaps routeToDomain and domainToRoute and then pick the domain/ route by key. It is just four elements so does not make any difference probably.

Reply:
Great suggestion. thanks